`python39Packages.pydyf` build failure on x86_64-linux as of `5a68a427`

## Issue description
Build of `python39Packages.pydyf` failed on x86_64-linux as of 5a68a4274d8ca3bbd0c455428a6d287daabbb6a6. This is currently breaking `python3Packages.wandb`.

```
error: builder for '/nix/store/im7zg3h4v509p0c1py9s2iq5w5ckdma3-python3.9-pydyf-0.1.2.drv' failed with exit code 1;
       last 10 log lines:
       > >                   assert 0, (
       >                         f'Pixel ({x}, {y}) in {name}: '
       >                         f'reference rgba{reference}, got rgba{value}')
       > E                   AssertionError: Pixel (0, 5) in test_text: reference rgba(255, 255, 255), got rgba(223, 223, 223)
       >
       > tests/__init__.py:69: AssertionError
       > =========================== short test summary info ============================
       > FAILED tests/test_pydyf.py::test_transform - AssertionError: Pixel (2, 4) in ...
       > FAILED tests/test_pydyf.py::test_text - AssertionError: Pixel (0, 5) in test_...
       > ========================= 2 failed, 20 passed in 1.81s =========================
       For full logs, run 'nix log /nix/store/im7zg3h4v509p0c1py9s2iq5w5ckdma3-python3.9-pydyf-0.1.2.drv'.
error: 1 dependencies of derivation '/nix/store/82hbk1p4885qbnp3mxwfzdxk8714p0fq-python3.9-weasyprint-54.3.drv' failed to build
error: 1 dependencies of derivation '/nix/store/d96xzj6rsmj8dvwszkknq7l1aa6hb74j-python3.9-trytond-6.4.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/y4ir53icxj2mc8lbf67jn276k09r42li-python3.9-sentry-sdk-1.5.11.drv' failed to build
error: 1 dependencies of derivation '/nix/store/98dmx1xcnbj67nb4k27wgzhpw10jrfd9-python3.9-wandb-0.12.16.drv' failed to build
```

[full build log](https://github.com/samuela/nixpkgs-upkeep/actions/runs/2297522381)

python3Packages.wandb maintainers cc: @samuela
python39Packages.pydyf maintainers cc: @Prillan
Other cc: n/a

### Steps to reproduce
1. Checkout nixpkgs at commit 5a68a4274d8ca3bbd0c455428a6d287daabbb6a6
2. Run `nix-build -A python39Packages.pydyf`

## Technical details
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.13.0-1022-azure, Ubuntu, 20.04.4 LTS (Focal Fossa), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.0`
 - nixpkgs: `/nix/store/xjqhdxv0qzhx0pn3izd767rw6qnpv9b0-source`
```

## Misc.
This issue was automatically generated by [nixpkgs-upkeep](https://github.com/samuela/nixpkgs-upkeep).
- [CI workflow](https://github.com/samuela/nixpkgs-upkeep/actions/runs/2297522381) that created this issue.
- Internal tags: -9053734745789360131 1692909980607733186

--- a/pkgs/development/python-modules/pydyf/default.nix
+++ b/pkgs/development/python-modules/pydyf/default.nix
@@ -1,5 +1,6 @@
 { lib
 , buildPythonPackage
+, fetchpatch
 , fetchPypi
 , isPy3k
 , pytestCheckHook
@@ -18,6 +19,15 @@ buildPythonPackage rec {
     pname = "pydyf";
     sha256 = "sha256-Hi9d5IF09QXeAlp9HnzwG73ZQiyoq5RReCvwDuF4YCw=";
   };
+
+  patches = [
+    # Fix tests for Ghostscript 9.56
+    # Remove after v0.1.3 has been released
+    (fetchpatch {
+      url = "https://github.com/CourtBouillon/pydyf/commit/d4c34823f1d15368753c9c26f7acc7a24fc2d979.patch";
+      sha256 = "sha256-2hHZW/q5CbStbpSJYbm3b23qKXANEb5jbPGQ83uHC+Q=";
+    })
+  ];
 
   postPatch = ''
     substituteInPlace pyproject.toml \

